### PR TITLE
[uss_qualifier] add scd scenario checking that CR deletion requires the correct OVN

### PR DESCRIPTION
**Pull request by Shastick**
The constraint reference (CR) equivalent of #761: ensure that a DSS will properly compare the passed OVN to the one in store before deleting a CR.

This is the second step towards #759

The PR includes an update to the validation of the `utm_implementation_us` scenario configuration: this scenario is for a jurisdiction that currently has no constraints processing. As such, it does not allow the qualifier to obtain the scopes relevant for constraint management, causing the new scenario to be skipped.

### CODE REVIEW

**Comment by Shastick** [comment]
Taking to draft: requires a last fix/adaptation for the `utm_implementation_us` configuration

File: monitoring/uss_qualifier/scenarios/astm/utm/dss/constraint_ref_simple.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+class CRSimple(TestScenario):
+    """
+    A scenario that checks that Constraint references cannot be deleted with the incorrect OVN.
+
+    TODO:
+     - enrich with test case to mutate with incorrect OVN (planned via #760)
+     - enrich with test case to delete with previous OVN (after mutation: add after #760)
+    """
+
+    CR_TYPE = register_resource_type(397, "Constraint Reference")
```
Comment (mickmis) [inline monitoring/uss_qualifier/scenarios/astm/utm/dss/constraint_ref_simple.py:37]: Update of the comment mentioning the next index?

**Review by mickmis — APPROVED** [review]
LGTM

**Review by mickmis — CHANGES_REQUESTED** [review]
I think two files got introduced in the PR but sohuldn't?

**Comment by Shastick** [comment]
> I think two files got introduced in the PR but sohuldn't?

Indeed. They have been removed.

**Review by mickmis — APPROVED** [review]

**Pull request merged**